BraveShared/Analytics/DAU.swift

```diff
@@ -19,14 +19,22 @@ public class DAU {
   private let pingRefreshDuration = 5.minutes
 
   /// We always use gregorian calendar for DAU pings. This also adds more anonymity to the server call.
-  fileprivate static var calendar: Calendar { return Calendar(identifier: .gregorian) }
+  fileprivate static var calendar: Calendar {
+    var cal = Calendar(identifier: .gregorian)
+    cal.locale = .init(identifier: "en_US")
+    if let timezone = TimeZone(abbreviation: "GMT") {
+      cal.timeZone = timezone
+    }
+    
+    return cal
+  }
 
   private var launchTimer: Timer?
-  private let today: Date
+  //private let today: Date
```

Comment:
Doesn't look like this is needed anymore?

Reply:
correct, will remove